sdk/cosmosdb/cosmos/test/public/integration/query.spec.ts

```diff
@@ -192,15 +192,22 @@ describe("Test Index metrics", function (this: Suite) {
 
   it("validate that index metrics are correct", async function () {
     const collectionId = "testCollection3";
-    const createdContainerSinglePartition = await setupContainer(
+    const containerDefinition = {
+      id: collectionId,
+    };
+    const containerOptions1 = { offerThroughput: 4000 };
```

Comment:
I see we are using `offerThroughput` here, how is it different from `throughput`?